chainer/functions/activation/hard_sigmoid.py

```diff
@@ -59,10 +59,21 @@ def hard_sigmoid(x):
         \\end{array} \\right.
 
     Args:
-        x (~chainer.Variable): Input variable.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_N)`-shaped float array.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_N)`-shaped float array.
+
+    .. admonition:: Example
+
+        >>> x = np.arange(-2, 3, 2).astype('f')
```

Comment:
How about using `x = np.array([-2.6, -1, 0, 1, 2.6])` instead? Because I thinks it's better to show all different cases that the above condition includes.